packages/client/hmi-client/src/components/data-explorer/search-item.vue

```diff
@@ -1,17 +1,61 @@
 <script setup lang="ts">
-import { XDDArticle } from '@/types/XDD';
+import { watch, ref, computed } from 'vue';
+import { XDDArticle, XDDExtractionType } from '@/types/XDD';
+import { Model } from '@/types/Model';
+import { Dataset } from '@/types/Dataset';
 import { isXDDArticle } from '@/utils/data-util';
 import IconAdd24 from '@carbon/icons-vue/es/add/24';
 import IconCheckmark24 from '@carbon/icons-vue/es/checkmark/24';
-import { ResultType } from '@/types/common';
+import IconOverflowMenuVertical24 from '@carbon/icons-vue/es/overflow-menu--vertical/24';
+import IconPDF16 from '@carbon/icons-vue/es/PDF/16';
+import IconLink16 from '@carbon/icons-vue/es/link/16';
+import IconChartLine16 from '@carbon/icons-vue/es/chart--line/16';
+import IconTable16 from '@carbon/icons-vue/es/table/16';
+import IconArrowLeft16 from '@carbon/icons-vue/es/arrow--left/16';
+import IconArrowRight16 from '@carbon/icons-vue/es/arrow--right/16';
```

Comment:
Now that we use PrimeVue, could you use their icons:
https://www.primefaces.org/primevue/icons